querydsl-core/src/main/java/com/querydsl/core/types/QBean.java

```diff
@@ -73,6 +78,10 @@
         return rv.build();
     }
 
+    private static boolean isAliasSplitted(Expression<?> expr) {
```

Comment:
I don't understand why an expression that is an instance of `FactoryExpression` or  `GroupExpression` is considered a split alias. I think the notion of splitting is confusing me.